misc-tools/dj_make_chroot.in

```diff
@@ -337,11 +316,7 @@ if [ -n "$DEBPROXY" ]; then
     echo "Acquire::http::Proxy \"$DEBPROXY\";" >> "$CHROOTDIR/etc/apt/apt.conf"
 fi
 
-mount -t proc proc "$CHROOTDIR/proc"
-mount -t sysfs sysfs "$CHROOTDIR/sys"
-
-# Required for some warning messages about writing to log files
-mount --bind /dev/pts "$CHROOTDIR/dev/pts"
+mount_host_special_files
```

Comment:
I'd call this `mount_filesystems` or so, or possibly `mount_host_special_filesystems`.